administration/console.md

```diff
@@ -15,15 +15,26 @@ The console offers the option to:
 
 ## Accessing the Console
 
-Accessing the console depends on the start mode of openHAB.
-The manually start using shell/batch script ends directly in the console.
+Accessing the console depends on how openHAB was started.
+When started manually using shell or batch script, openHAB naturally transitions directly to the console.
+When started as a service, the console can be reached via local SSH.  
```

Comment:
Even explained this good, users might not know which path they are on. e.g openHABian user might not know the meaning of "service". Maybe it would be wise to mention the "installed from package repository" here somehow.

I'd remove "local" here and instead link SSH, users might not know what it is.